HotelPriceScout/Data/Interface/Dashbord.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+namespace HotelPriceScout.Data.Interface
+{
+    public class Dashboard
+    {
+        public string monthName = "";
+        public DateTime monthEnd;
+        public int monthsAway = 0;
+        public int numDummyColumn = 0;
+        public int year = 0;
+        public int month = 0;
+        public int DayClicked = 0;
```

Comment:
overvej at bruge default istedet for 0. Default forklarer mere at det er en værdi vi ikke har sat endnu. For eksempel ville man ved en konto's mængde af penge bruge 0, da man kun har 0 kr. I vores tilfælde er det vel bare en værdi vi assigner senere